lightkurve/targetpixelfile.py

```diff
@@ -41,6 +44,110 @@ def __init__(self, path, quality_bitmask='default', targetid=None, **kwargs):
         self.quality_mask = self._quality_mask(quality_bitmask)
         self.targetid = targetid
 
+
+    def get_sources(self, catalog=None, magnitude_limit=18, dist_tolerance=3):
+        """
+        Returns a table of stars that are centered on the target
+        of the tpf file. Current catalog supported are KIC, EPIC and Gaia DR2.
+
+        Parameters
+        -----------
+        catalog: 'KIC', 'EPIC' or 'Gaia'
+            Catalog to query. For Kepler target pixel files the default is 'KIC',
+            for K2 target pixel files the default is 'EPIC'.
+        magnitude_limit : int or float
+            Limit the returned magnitudes to only stars brighter than magnitude_limit.
+            Default 18 for both Kepmag and Gmag.
+        dist_tolerance: float
+            Maximum distance (in arcseconds) a source may be separated from the edge
+            of the target pixel file.
+
+        Returns
+        -------
+        result : astropy.table
+            Astropy table with the following columns
+            ID : Catalog ID from catalog.
+            RA: Right ascension [degrees] (KIC & EPIC)
```

Comment:
If you have changed the RA/Dec from the catalog to the current epoch you MUST state that somewhere in the docstring. Perhaps here in the RA/Dec